lib/TimespanLogging/src/core.jl

```diff
@@ -233,28 +233,35 @@ function write_event(ml::MultiEventLog, event::Event)
     end
 end
 
-function get_logs!(ml::MultiEventLog; only_local=false)
-    logs = Dict{Int,Dict{Symbol,Vector}}()
-    wkrs = only_local ? myid() : procs()
-    # FIXME: Log this logic
-    @sync for p in wkrs
-        @async begin
-            logs[p] = remotecall_fetch(p, ml) do ml
-                mls = get_state(ml)
-                lock(event_log_lock) do
-                    sublogs = Dict{Symbol,Vector}()
-                    for name in keys(mls.consumers)
-                        sublogs[name] = mls.consumer_logs[name]
-                        mls.consumer_logs[name] = []
+function get_logs!(ml::MultiEventLog; only_local=true)
+    if only_local
+        # Only return logs from current process
+        mls = get_state(ml)
+        return mls.consumer_logs
```

Comment:
We need to make a copy of these logs and then empty each vector contained in `mls.consumer_logs`, as the API of `get_logs!` is to remove the old logs from `mls.consumer_logs` so that future calls only include new logs.

Reply:
okay will fix it